evalml/tests/data_checks_tests/test_id_columns_data_check.py

```diff
@@ -335,13 +335,15 @@ def test_unidentified_first_col_primary_key(
     ]
 
     X = X.rename(columns={"col_1_id": "col_1"})
-    if input_type == "integer":
+    if input_type == "Integer":
         X.at[0, "col_1"] = 0
-    elif input_type == "integer_nullable":
+    elif input_type == "IntegerNullable":
         X.at[0, "col_1"] = 0
-    elif input_type == "double":
+    elif input_type == "Double":
         X.at[0, "col_1"] = 0.0
-    elif input_type == "string":
```

Comment:
Is `Text` replacing `string` now? Should we also do an IF for `Unknown`?

Reply:
The variable `input_type` has no bearing on the type of the first column. That is determined within the block of code within the IF statement. I just changed the name because it was consistent with the rest of the names as being logical types. I could change the name to `Unknown` actually because that is what the logical type of the first column of the data within that IF block is. There is no `Text` Woodwork logical type so I cannot manually set the type of the first column to be that. However, the UI somehow is able to make the logical type of the first column as `Text`

Reply:
Got it, that makes sense!